### ISSUE TITLE
Curl of gradient?

### ISSUE THREAD

**Issue by piannucci** [HomerReid/buff-em-documentation#1]
This can't be right -- tex/buff-em.tex, line 154.  You must mean curl of curl?

**Comment by piannucci** [comment]
Also, line 200, RHS should be LHS.

**Comment by HomerReid** [comment]
I agree that the nabla \times nabla notation is confusing. I revised it.

Good catch on line 200. I fixed it.

Feel free to keep this issue alive with new comments or questions, or close the issue if satisfied.
